### PR TITLE
fix: Add `user` filed to get problem set grades API

### PR DESCRIPTION
**Pull request by qianxi66**
实现#132 的
Class管理员，查询该Class中某一ProblemSet的全部用户成绩 需求。
继承#133pr的代码写的。

### CODE REVIEW

File: base/utils/problem_set.go [hunk 1]
```diff
@@ -127,11 +129,16 @@ func CreateEmptyGrades(problemSet *models.ProblemSet) error {
...
+	emptyDetail, err := json.Marshal(detail) // 将map转换为JSON格式
 	if err != nil {
+		// 如果转换失败，记录错误并返回
+		log.Errorf("Error marshalling grade detail for empty grade: %v", err)
 		return errors.Wrap(err, "could not marshal grade detail for empty grade")
 	}
 
+	// 打印转换后的JSON日志
+	log.Errorf("Empty detail JSON: %s", emptyDetail)
+
```
Comment (leoleoasd) [inline base/utils/problem_set.go:141]: Please write comments in English

**Review by leoleoasd — CHANGES_REQUESTED** [review]

**Review by leoleoasd — APPROVED** [review]
LGTM

**Comment by qianxi66** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by leoleoasd — APPROVED** [review]
LGTM

**Pull request merged**